### PR TITLE
Modification of the test.yml file in .github

### PR DESCRIPTION
**Pull request by devrulls**
Modification of the test.yml file in .github
creation de dockerfilein webapp .
modification of streamlit version==1.8.1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+clean:
+	rm -rf venv
+	rm -rf *.egg-info
+	find . | grep -E "__pycache__|\.pyc|\.pyo$$|.pytest_cache|.mypy_cache" | xargs rm -rf
+
+.PHONY: test
+test:
+	python -m black pi_server
+	python -m mypy pi_server
+	python -u -m pytest -vv
```
Comment (Saxamos) [inline Makefile]: Why did you put this file back?

File: pi_server/pi_server/views/routes.py [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-    time.sleep(1)
-
-    #Finish at 180°
-    pwm.ChangeDutyCycle(convert_angle_to_servo_rotation(180))
-    time.sleep(1)
-
-    #Close GPIO & cleanup
-    pwm.stop()
-    GPIO.cleanup()
+from pi_server import app
```
Comment (Saxamos) [inline pi_server/pi_server/views/routes.py:1]: Mmmh I don't understand why there are so many changes in the diff view. Did you change smt in all those files?

**Review by Saxamos — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (devrulls) [reply Makefile]: I don't know what happened, normally all those changes were already applied. =s
I'm going to delete this file.

File: pi_server/pi_server/views/routes.py (hunk 2 above)
Comment (devrulls) [reply pi_server/pi_server/views/routes.py:1]: I don't understand, these modifications were made with the PR from the raspberry.
it's the same code. =s

**Review by Saxamos — APPROVED** [review]

**Pull request merged**